tilauskasittely/tarkistarivi.inc

```diff
@@ -92,6 +92,31 @@ if ($trow['panttitili'] == 'K' and in_array($toim, array('PIKATILAUS', 'RIVISYOT
   }
 }
 
+$_onko_myynti = in_array($toim, array('PIKATILAUS','RIVISYOTTO'));
+$_onko_param = ($yhtiorow['myynti_jt_huom'] == 'K');
+
+if ($_onko_param and $_onko_myynti and $kukarow['extranet'] == '') {
+  $query = "SELECT *
+            FROM lasku
+            JOIN tilausrivi ON (tilausrivi.yhtio = lasku.yhtio
+              AND tilausrivi.otunnus = lasku.tunnus
+              AND tilausrivi.tuoteno = '{$row['tuoteno']}'
+              AND tilausrivi.tyyppi = 'L'
+              AND tilausrivi.var = 'J'
+              AND tilausrivi.jt != 0
+              AND tilausrivi.tunnus != '{$row['tunnus']}'
+            )
+            WHERE lasku.yhtio = '{$kukarow['yhtio']}'
+            AND lasku.liitostunnus = '{$laskurow['liitostunnus']}'
+            AND lasku.tunnus != '{$laskurow['tunnus']}'
+            AND lasku.tila = 'N'";
```

Comment:
@samiluukkonen Onko joku spessusyy miks on näin tiukka rajaus -->  lasku.tila = 'N' ? Eli jos JT-rivi onko tulosteulla, kerätyllä, tai toimitetulla tilauksella, niin ei näytetän tässä?